reV/bespoke/place_turbines.py

```diff
@@ -329,6 +329,35 @@ def place_turbines(self, **kwargs):
         self.initialize_packing()
         self.optimize(**kwargs)
 
+    def capital_cost_per_kw_at(self, capacity_mw):
```

Comment:
What's up with the `_at` suffix? "At the input capacity"? Kind of funky naming IMO but fine if you like it.

Reply:
Yea I guess the thought was to make it read like English: `capital_cost_per_kw_at(200)`.
Happy to change it to  `capital_cost_per_kw`, I don't have a strong preference here

Reply:
Yeah gotcha. This is an interesting one haha I haven't thought to do this. I could go either way!